scilpy/reconst/utils.py

```diff
@@ -46,12 +46,14 @@ def _honor_authorsnames_sh_basis(sh_basis_type):
     return sh_basis
 
 
-def get_b_matrix(order, sphere, sh_basis_type, return_all=False):
+def get_b_matrix(order, sphere, sh_basis_type, return_all=False,
```

Comment:
I do not think we actually need this function anymore, it just calls DIPY's method, no?

Reply:
I thought the same thing... @arnaudbore ?

Reply:
Can you show me what it will look like ? It seems that get_b_matrix is doing a little bit more than one call to DIPY's method.